### ISSUE TITLE
Can we use CityEngine to add buildings and vegetation?

### ISSUE THREAD

**Issue by pengzhenghao** [decisionforce/pgdrive#190]
Link: https://nv-tlabs.github.io/NTG/

I think that could be a good enhancement for PGDrive! Especially for RGB rendering!

But it seems that the CityEngine is a commercial software? That could harm the open-source feature of PGDrive!

<img width="924" alt="image" src="https://user-images.githubusercontent.com/22206995/103202953-d5d8a380-492e-11eb-9434-651aa3eb29a3.png">

<img width="907" alt="image" src="https://user-images.githubusercontent.com/22206995/103202985-e2f59280-492e-11eb-834d-bfc0c2ee466e.png">

<img width="821" alt="image" src="https://user-images.githubusercontent.com/22206995/103203008-f143ae80-492e-11eb-918a-6c017b43e579.png">

**Comment by lqy0057** [comment]
Yes, it is commercial software. How to get various models is the most difficult part. It seems that it generates models by procedural generation too. Panda3d also supports procedural generating models. After surveying this field and learning from CItyEngine, maybe we can try this in the future.

**Comment by pengzhenghao** [comment]
> Yes, it is commercial software. How to get various models is the most difficult part. It seems that it generates models by procedural generation too. Panda3d also supports procedural generating models. After surveying this field and learning from CItyEngine, maybe we can try this in the future.


"Panda3d also supports procedural generating models" what does this mean? Can we procedural generate vehicle model now for a start?

**Comment by zhoubolei** [comment]
Take a look at this recent project from NYU: https://github.com/ai4ce/RealCity3D
https://openreview.net/pdf?id=VjFn0L5T7NU. I know the PI and potentially we can have some collaboration
